main.js

```diff
@@ -28,6 +30,7 @@ module.exports = function(options) {
 		withAnonMiddleware: false,
 		withBackendAuthentication: false,
 		withRequestTracing: false,
+		hasHeadCss: false,
```

Comment:
yeah i agree with that.  quite nice to have an incentive to split things out for a change :]